ocaml/client_records/records.ml

```diff
@@ -536,6 +536,7 @@ let pool_record rpc session_id pool =
 				();
 			make_field ~name:"cpu_info" ~get:(fun () -> Record_util.s2sm_to_string "; " (x ()).API.pool_cpu_info) ~get_map:(fun () -> (x ()).API.pool_cpu_info) ();
 			make_field ~name:"policy-no-vendor-device" ~get:(fun () -> string_of_bool (x ()).API.pool_policy_no_vendor_device) ~set:(fun s -> Client.Pool.set_policy_no_vendor_device rpc session_id pool (safe_bool_of_string "policy-no-vendor-device" s)) ();
+			make_field ~name:"live-patching-enable"  ~get:(fun () -> string_of_bool (x ()).API.pool_live_patching_enable) ~set:(fun s -> Client.Pool.set_live_patching_enable rpc session_id pool (safe_bool_of_string "live-patch-enable" s)) ();
```

Comment:
For consistency with "wlb-enabled" and "ha-enabled", this should be "live-patching-enabled".